### PR TITLE
fix the member counter

### PR DESCRIPTION
**Pull request by CircuitSacul**
This will make it so that the member count channel will actually update over time.

### CODE REVIEW

**Review by Endercheif — APPROVED** [review]
👍

**Review by qexat — APPROVED** [review]
Looks fine to me

File: mcodingbot/tasks/stats.py [hunk 1]
```diff
@@ -46,16 +46,25 @@ async def update_channels(bot: Bot) -> None:
...
     if not guild:
         return LOGGER.warning(
             "Couldn't find mCoding guild, not updating member count."
         )
-    if not member_channel:
-        return LOGGER.warning("No member count channel to update stats for.")
-    member_count = guild.member_count
-    if member_count is None:
-        return LOGGER.error("Cached guild didn't contain member count.")
+    guild_aprox_members = guild.member_count
```
Comment (Lunarmagpie) [inline mcodingbot/tasks/stats.py:58]: I don't think the `guild_approx_member` variable increases clarity

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: mcodingbot/tasks/stats.py (hunk 1 above)
Comment (CircuitSacul) [reply mcodingbot/tasks/stats.py:58]: It has to be stored so I can check if it's None, but still make mypy happy.

Comment (qexat) [reply mcodingbot/tasks/stats.py:58]: There is also a little typo in the name.

Comment (Lunarmagpie) [reply mcodingbot/tasks/stats.py:58]: Mypy moment

Comment (Lunarmagpie) [inline mcodingbot/tasks/stats.py:58]:
```suggestion
    guild_approx_members = guild.member_count
```

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

**Review by Lunarmagpie — APPROVED** [review]

**Comment by qexat** [comment]
:rocket:

**Pull request merged**